cmd/gobgpd/main.go

```diff
@@ -205,6 +206,14 @@ func main() {
 	bgpServer := server.NewBgpServer(server.GrpcListenAddress(opts.GrpcHosts), server.GrpcOption(grpcOpts))
 	go bgpServer.Serve()
 
+	if status, err := daemon.SdNotify(false, daemon.SdNotifyReady); !status {
+		if err != nil {
```

Comment:
If systemd is not available on a system, we get a warning. I don't think that it's reasonable. Can you add an option to enable this feature?

Reply:
I added an option and updated the example of the systemd service